ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/add/steps/DataMapper.java

```diff
@@ -137,6 +137,12 @@ public void createMapping(String source, String target) {
      * This method opens all collection mappings we can find on a data mapper
      */
     public void openDataMapperCollectionElement() {
+        try {
+            log.info("Waiting for Data Mapper to fully load before opening all collection mappings");
+            Thread.sleep(3000);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
         $$(Element.MAPPER_COLLECTION_ICON).forEach(e -> {
```

Comment:
If you are waiting for something, do not use static sleep but dynamic TestUtils.waitFor...